test/jdk/javax/swing/JTabbedPane/bug4666224.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2001, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.Dimension;
+import java.awt.Point;
+import java.awt.event.ActionEvent;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import javax.swing.AbstractAction;
+import javax.swing.ActionMap;
+import javax.swing.InputMap;
+import javax.swing.JComponent;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.JTabbedPane;
+import javax.swing.LookAndFeel;
+import javax.swing.SwingUtilities;
+
+/* @test
+ * @bug 4666224
+ * @summary Tests that focus indicator is painted properly in JTabbedPane
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual bug4666224
+ */
+
+public class bug4666224 {
+    private static JTabbedPane tabPane;
+    private static JFrame frame;
+
+    private static final String INSTRUCTIONS = """
+                ON ALL PLATFORMS
+                    1. Click on any of the tabs, focus indicator is visible.
+                    2. Lose focus on the window by clicking on some other window.
```

Comment:
Lose focus?

Reply:
Yeah, it means the current focused windows should not be focused.

Reply:
I was getting confused whether it should be lose or loose in this context?

Reply:
I think `Lose` is ok.